<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupDialog</class>
 <widget class="QDialog" name="SetupDialog">
  <property name="windowModality">
   <enum>Qt::WindowModality::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>State setup</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget_10" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QRadioButton" name="useDistributionRadioButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Use distribution</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="manualSetupRadioButton">
        <property name="text">
         <string>Manual setup</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_7" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1">
      <item>
       <widget class="QWidget" name="distributionWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Discs</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="numberOfDiscsSpinBox">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>5000</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="discTypeDistributionTableView"/>
         </item>
         <item>
          <widget class="QWidget" name="widget_13" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QPushButton" name="addDistributionEntryPushButton">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>75</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="clearDistributionPushButton">
              <property name="maximumSize">
               <size>
                <width>75</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="manualWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTableView" name="discsTableView"/>
         </item>
         <item>
          <widget class="QWidget" name="widget_14" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QPushButton" name="addDiscPushButton">
              <property name="maximumSize">
               <size>
                <width>75</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="clearDiscsPushButton">
              <property name="maximumSize">
               <size>
                <width>75</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_11" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Timestep [Î¼s]</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="timeStepSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="singleStep">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Timescale</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="timeScaleDoubleSpinBox">
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="singleStep">
            <double>1.000000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_9" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Cell height</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="cellHeightSpinBox"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_6" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Cell width</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="cellWidthSpinBox"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_12" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>GUI FPS</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="GUIFPSSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_8" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="okPushButton">
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelPushButton">
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
